packages/modern-poalim-scraper/src/scrapers/types/cal/get-card-transactions-details.ts

```diff
@@ -0,0 +1,155 @@
+export interface CalGetCardTransactionsDetailsResponse {
```

Comment:
existing scrapers run strict validation against the new data, to make sure we cover all variations and track API changes of the data source.
We use json schemas for that. I think here it's only TS type-safety right?

Reply:
Correct. I can validate all input first.

Reply:
Handled with zod